packages/application-shell/src/components/application-shell/application-shell.js

```diff
@@ -319,11 +317,6 @@ export const RestrictedApplication = props => (
                                 path="/:projectKey"
                                 render={routerProps => (
                                   <React.Fragment>
-                                    <ReconfigureFlopFlip
-                                      user={getFlopflipReconfiguration(
-                                        routerProps.match.params.projectKey
-                                      )}
-                                    />
```

Comment:
Cause this component and `ConfigureFlopflip` rendered by `SetupFlopFlipProvider` end up in a "fight" overwriting each others `adapterArgs`. There are props to influence how flopflip can merge args but I don't think it's worth having this complexity.